achievements.html

```diff
@@ -1,45 +1,45 @@
 <!DOCTYPE html>
-<html>
+<html lang = "en">
 <head>
+	<meta charset="utf-8"/>
 	<title>GirlScript Foundation-Chennai Chapter | Achievements</title>
```

Comment:
Make it GirlScript Website Boilerplate

Reply:
@smaranjitghose the title? I'm so sorry if I am wrong, I am an absolute beginner, just trying to learn all these things.

Reply:
Yes, please